include/llvm/Support/WinMacros.h

```diff
@@ -52,7 +50,7 @@
 // TODO(ehsann): This is an unsafe conversion.
 #define UIntToInt(uint_arg, int_ptr_arg) *int_ptr_arg = uint_arg
 
-#define INVALID_HANDLE_VALUE (void*)nullptr
+#define INVALID_HANDLE_VALUE (void *)nullptr
```

Comment:
Just as an aside, the real value for INVALID_HANDLE_VALUE on Windows is a void pointer with value -1. I don't know if there are any macros that rely on that sign bit being set or not.